hera_mc/cm_dataview.py

```diff
@@ -0,0 +1,101 @@
+#! /usr/bin/env python
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2017 the HERA Collaboration
+# Licensed under the 2-clause BSD license.
+
+"""Allows some different data views of cm database.
+Currently this is only to look at the full connections of parts looped over time,
+either via db calls or pre-written files.
+"""
+
+from __future__ import absolute_import, print_function
+from hera_mc import mc, cm_utils, cm_revisions, cm_hookup
+from astropy.time import Time, TimeDelta
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+def write_file(filename, parts_list, fc_map, output='flag'):
+    from hera_mc import cm_hookup
+    hu = cm_hookup.Hookup(session)
```

Comment:
It doesn't look like 'session' will be defined here, and it's not passed in...
edit: I see, it's a global variable you define in a later function. I'm not sure I understand the value of the global variable here, rather than just passing it in, I think generally avoiding globals is a good thing. But if there's a really good reason to use a global, it really should be defined earlier in the file than it's used for readability.

In general, I think that functions should occur in a file in an order such that you can always look earlier in the file to find something that is being called by a later function.